tests/schemas/api_v2/fees.schema.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2021 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import Joi from 'joi';
+
+const feeEstimateSchema = {
+	feeEstimatePerByte: Joi.object({
+		low: Joi.number().min(0).required(),
+		medium: Joi.number().min(0).required(),
+		high: Joi.number().min(0).required(),
+	}).required(),
+	baseFeeById: Joi.object({
+		'5:0': Joi.string().required(),
```

Comment:
Rather than hardcoding the object, please use the Joi `object.pattern` instead.

Ref: https://github.com/sideway/joi/blob/master/API.md#objectpatternpattern-schema-options